docs/wiki/modules/llm-call.md

```
@@ -0,0 +1,44 @@
+
+All paths relative to `backend/app/`.
+
+## Routes
+- `api/routes/llm.py` — `POST /llm/call`
+- `api/routes/llm_chain.py` — chains
+- `api/routes/llm_sts.py` — speech-to-speech
+- `api/routes/config/config.py`, `api/routes/config/version.py` — saved config CRUD + versions
+- `api/routes/guardrails.py` — guardrail validators
+
+## Tables (SQLModel)
+| Table | Model |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)